config.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  play:
+    url: https://play.openpolicyagent.org
+
+bundles:
+  authz:
+    service: play
+    resource: bundles/WfbLy2DbiF
+
+decision_logs:
+  console: true
+
+default_decision: /http/example/authz/allow
```

Comment:
Is this file used anywhere ? If yes then where is the policy corresponding to the  `default_decision` ?